app/assets/javascripts/compute_resource.js

```diff
@@ -4,7 +4,7 @@ $(function() {
     var url = $(this).attr('data-url');
     $(this).load(url + ' table', function(response, status, xhr) {
       if (status == "error") {
-        $(this).closest(".tab-content").find("#spinner").html(Jed.sprintf(__("There was an error listing VMs: %(status)s %(statusText)s"), {status: xhr.status, statusText: xhr.statusText}));
+        $(this).html(Jed.sprintf(__("There was an error listing VMs: %(status)s %(statusText)s"), {status: xhr.status, statusText: xhr.statusText}));
```

Comment:
To further illustrate:
![image](https://cloud.githubusercontent.com/assets/4107560/22330620/772cbd6a-e3c7-11e6-8391-e8a91da17085.png)